docs/application/tizen-studio/native-tools/batt-historian.md

```diff
@@ -0,0 +1,157 @@
+
+### View and Analyze Battery Data
+
+To view and analyze your data, follow these steps:
+
+1. In Battery Historian Main Window, click **Browse**.
+
+2. Select the appropriate bug report file.
+
+3. Click **Submit.**
+
+Battery Historian takes a few seconds to analyze the data in the file. After the analysis is done, a battery stats chart appears:
+
+**Figure:** Battery Historian Chart
+
+![](./media/graph1.png)
+
+The Battery Historian chart graph illustrates the battery consumption events over time, where usage of the various device components correlates to battery consumption.
+
+#### Interpret the Graph
+
+ - The vertical axis represents the battery percentage, while the horizontal axis represents time.
+ - The sloping black bar indicates your battery percentage level throughout the day.
+ - The colored line segments represent the various apps, services, and other elements that have used battery life.
+   - For instance, a green bar denotes an active time for WiFi.
+ - You can add more metrics, by clicking on Add Metric and the corresponding metric will appear on the chart, some of the available metrics are Voltage, Charging status, and so on.
+ - The chart generated is intuitive, hovering over any metric name in the first column shows you more information about that metric.
+
+**Figure**: Additional Stats
+
+![](./media/graph2.png)
+
+The **System Stats** tab includes overall system statistics, such as screen on time, WiFi idle time, and so on. It gives an overview of what is happening with your Tizen device. For example, you can analyze the impact of a particular application on the battery during its use.
+
+### Compare Bug Reports 
+
+The battery Historian also allows comparative analysis of two bug reports. The analysis can be viewed in the graphs that highlights the key differences in the battery-related metrics. 
+
+It also helps you to compare and contrast the metrics for battery consumption, and explains differences or similarities for any two key battery metrics of the Tizen device. 
+
+The compare feature of the Battery Historian tool helps you understand which metric is responsible for battery drainage, and diagnose the problem more efficiently. 
+
+To compare the bug report files, follow these steps:
+
+1. In Battery Historian Main Window, click **Switch to Bug Report Comparison**.
+
+    The bug report compare window appears: 
+
+    ![](./media/compare1.png)
+
+    The bug report compare window comprises of the following components: 
+      - **Browse**: This component allows you to browse and select a bug report file.
+         - The window has two **Browse** buttons to allow you to upload two bug report files. 
+
+2. Click **Browse.**
+   
+   Select the first bug report file. 
+   
+3. Click **Browse.** 
+   
+   Select the second bug report file.  
+
+4. Click **Submit.**
+
+Battery Historian takes a few seconds to analyze the data in both the files, and a comparison chart appears: 
+
+![](./media/compare2.png)
```

Comment:
**Suggestion :** 
If we can update the image, how about using another?

This image is inserted to show the comparison result, but the File 2 rarely has value to show up in the comparison result.

**Question :** 
In the middle of the image, I can see the "Screen Usage Data".
I'm not sure whether it is the UI of the tool or inserted text at the image.
If the second part is not the UI view of the tool but the added part,
please separate the image and insert the figure caption to describe the image.